docs/operators/operator-node/maintenance/troubleshooting.md

```diff
@@ -349,10 +210,205 @@ CONFIG_PATH=/config.yaml -p 13001:13001 -p 12001:12001/udp -p 15000:15000 \
+* `inbound_conns` should be at least 20% of the peers (though not an exact number, this is a good indication of healthy connections from the node)
+
+Below, an example of the same report, from a **node in bad state**:
+
+```json
+{
+  "p2p": "bad: not enough connected peers",
+  "beacon_node": "good",
+  "execution_node": "good",
+  "event_syncer": "good",
+  "advanced": {
+    "peers": 5,
+    "inbound_conns": 0,
+    "outbound_conns": 4,
+    "p2p_listen_addresses": [
+      "tcp://<X.Y.W.Z>:13004",
+      "udp://<X.Y.W.Z>:12004"
+    ]
+  }
+}
+```
 </details>
 
+## SSV-Pulse benchmarking tool
+
+**Before using this tool** make sure to open [SSV Node Health Endpoint](#ssv-node-health-endpoint).
+
+Benchmark analyzes SSV, Consensus, and Execution Nodes at the same time. If curious, you can find more details on [ssv-pulse GitHub page](https://github.com/ssvlabs/ssv-pulse).
 
+To use this tool you can use docker compose or a docker command below:
+<Tabs>
+  <TabItem value="docker-compose" label="docker compose">
+
+If you used docker compose to run your SSV node — add the following part after the `services:` to your `docker-compose.yml` file:
+
+```yaml
+services:
+  ssv-pulse:
+    container_name: ssv-pulse
+    image: ghcr.io/ssvlabs/ssv-pulse:latest
+    command: 
+      - 'benchmark'
+      - '--consensus-addr=<YOUR_ADDRESS_HERE>' # Change to Consensus Node's address, e.g. http://lighthouse:5052
+      - '--execution-addr=<YOUR_ADDRESS_HERE>' # Change to Execution Node's address, e.g. http://geth:8545
+      - '--ssv-addr=<YOUR_ADDRESS_HERE>' # Change to SSV Node's address, e.g. http://ssv_node:16000
+      - '--duration=60m'
+      # - '--network=holesky' # Add this if you run a Holesky Node
+      # - '--platform linux/arm64' # Add this if you run on an arm64 machine
+    networks:
+      - host # This allows ssv-pulse to access any docker container hosted on the machine
+            # Alternatively, you can use your existing network instead. Check with docker network ls.
+    pull_policy: always
+```
+
+Then run `docker compose up ssv-pulse` to run the benchmark tool.
+
+  </TabItem>
+  <TabItem value="docker-run" label="docker run">
+
+  Use the following command to run the benchmark tool:
+
+  ```bash
+  docker run --rm --network=host --pull=always --name ssv-pulse \
+  ghcr.io/ssvlabs/ssv-pulse:latest benchmark \
+  --consensus-addr=REPLACE_WITH_ADDR \
+  --execution-addr=REPLACE_WITH_ADDR \
+  --ssv-addr=REPLACE_WITH_ADDR \
+  --duration=60m
+  ```
+
+  Replace the various addresses with the respective endpoints, e.g. `http://lighthouse:5052`, `http://geth:8545`, and SSV `http://ssv_node:16000` (or your other SSVAPIPort).
+
+  - If you run a Holesky Node you should add `--network=holesky` to the command.
+  - If you run this on a arm64 machine you should add `--platform linux/arm64` to the command.
+  </TabItem>
+</Tabs>
+
+<details>
+
+<summary><strong>How to interpret Benchmark results?</strong></summary>
```

Comment:
I like the current notes better, but at this point, this is not a section on how to **correctly interpret results**. This is rather a note on false alarms. So maybe change the title of this spoiler to something like "Please beware...".